README.md

```diff
@@ -1,79 +1,31 @@
-# Extra-CerebNEST
+# extra_cereb_nest
 
-Tested with:
+pip installable NEST extension module, generated by pipnest.
 
-*branch master*: Ubuntu 18.04 and Red Hat Enterprise Linux Server 7.7 (Maipo), Python 3.7.3 and NEST Release 2.18.0
+## Installation instructions
 
+### Using pip
 
-### Installation instructions
+When installed through pip the module is built for you with CMake outside of the python
+package folder, this means that pip cannot track the real installation status. To
+uninstall and reinstall see [Uninstallation](Uninstallation).
 
-0. Install NEST following the instructions provided here (http://www.nest-simulator.org/)
+0. Install NEST following the instructions provided here (https://www.nest-simulator.org/)
 
-1. Export an Environment Variable containing the installation directory of NEST. E.g.:
-```
-export NEST_INSTALL_DIR=$HOME/nest-simulator-install
-```
-2. Clone this GitHub Repository in a directory outside NEST source and build directories. E.g.:
-```
-cd $HOME
-git clone https://github.com/dbbs-lab/extra-cereb-nest/
-```
-3. Move to Extra-CerebNEST directory and create a new folder where you will build the extension module
-```
-mkdir $HOME/extra-cereb-nest-build
-cd $HOME/extra-cereb-nest-build
-```
-4. Run the following CMake command (Tested with CMake 3.2.2)
-```
-cmake -Dwith-nest=${NEST_INSTALL_DIR}/bin/nest-config $HOME/extra-cereb-nest-build
-```
-
-The resulting output should be something similar to:
-> [...]
->-------------------------------------------------------
->
->You can now build and install 'extracerebmodule' using
->  make
->  make install
->
->The library file libextracerebmodule.so will be installed to
->  /home/alberto/nest-simulator-install/lib/nest
->Help files will be installed to
->  /home/alberto/nest-simulator-install/share/doc/nest
->
->The module can be loaded into NEST using
->  (extracerebmodule) Install        (in SLI)
->  nest.Install('extracerebmodule')  (in PyNEST)
->
->The module will be installed into a non-default location!
->Make sure to set the environment variables:
->  export NEST_MODULE_PATH=/home/alberto/nest-simulator-install/lib/nest:$NEST_MODULE_PATH
->  export SLI_PATH=/home/alberto/nest-simulator-install/share/nest/sli:$SLI_PATH
-> [...]
+1. Install using pip
 
-5. Execute the last two lines, generated by the CMake command. 
 ```
-export NEST_MODULE_PATH=/home/alberto/nest-simulator-install/lib/nest:$NEST_MODULE_PATH
-export SLI_PATH=/home/alberto/nest-simulator-install/share/nest/sli:$SLI_PATH
```

Comment:
Is this part taken into account?

Reply:
No what needs to happen with that?

Reply:
to let NEST know where models are installed. When you compile and install a NEST module a message telling you to do that appears at the end of configuration.

Maybe with pipnest this is not needed, but you should check :-)

Reply:
Thanks for the advise; I'll check by updating the travis pipeline to skip this step. If a clean VM can load the module without this then I think we can ignore it.